gomoku: add page (#23353)
tldr-pages/tldr: pages/common/gomoku.md

# gomoku

> Gomoku, also known as five in a row, is a 2 player strategy game.

> > Your goal is to be the first one to get 5 in a row.

Reviewer: ```suggestion
> Play the classic 5-in-a-row strategy board game.
```

> More information: <https://manned.org/man/gomoku>.

- Start the game:

`gomoku`

- Move the cursor:

> `{{[<h>|<j>|<k>|<l>|<ArrowLeft>|<ArrowDown>|<ArrowUp>|<ArrowRight>]}}`

Reviewer: ```suggestion
`{{<h>|<j>|<k>|<l>|<ArrowLeft>|<ArrowDown>|<ArrowUp>|<ArrowRight>}}`
```

- Select the cell:

`<Space>`

> - Quit:

Reviewer: ```suggestion
- Quit the game:
```
